gobblin-metrics/src/main/java/gobblin/metrics/OutputStreamReporter.java

```diff
@@ -127,7 +128,11 @@ public T outputTo(PrintStream output) {
      * @return {@code this}
      */
     public T outputTo(OutputStream stream) {
-      this.output = new PrintStream(stream);
+      try {
+        this.output = new PrintStream(stream, false, "UTF-8");
```

Comment:
You can use Guava's Charsets.UTF-8 to be consistent with the rest of the Gobblin code.